modules/client-configuration/pages/supported-features-oracle.adoc

```diff
@@ -0,0 +1,66 @@
+[[supported-features-oracle]]
+= Supported Oracle Features
+
+
+This table lists the availability of various features on {oracle} clients.
+
+The icons in this table indicate:
+
+* icon:check[role="green"] the feature is available on both traditional and Salt clients
+* icon:times[role="danger"] the feature is not available
+* icon:question[role="gray"] the feature is under consideration, and may or may not be made available at a later date
+* ``Traditional`` the feature is supported only on traditional clients
+* ``Salt`` the feature is supported only on Salt clients
+
+
+[cols="1,1", options="header"]
+.Supported Features on {oracle} Operating Systems
+|===
+| Feature | {oracle}{nbsp}8
+| Client | icon:question[role="gray"] (plain CentOS) / ``Salt``
+| Operating system packages  | ``Salt``
+| Registration | ``Salt``
+| Install packages | ``Salt``
+| Apply patches (requires CVE ID) | ``Salt`` (third-party service required for errata)
+| Remote commands | ``Salt``
+| System package states | ``Salt``
+| System custom states | ``Salt``
+| Group custom states | ``Salt``
+| Organization custom states | ``Salt``
+| System set manager (SSM) | ``Salt``
+| Service pack migration | icon:times[role="danger"]
+| Basic Virtual Guest Management icon:asterisk[role="none"] | ``Salt``
+| Advanced Virtual Guest Management icon:asterisk[role="none"] | ``Salt``
+| Virtual Guest Installation (Kickstart), as Host OS | icon:times[role="danger"]
+| Virtual Guest Installation (image template), as Host OS | ``Salt``
+| System deployment (PXE/Kickstart) | ``Salt``
+| System redeployment (Kickstart) | ``Salt``
+| Contact methods | ``Salt``: ZeroMQ, Salt-SSH
+| Works with {productname} Proxy | ``Salt``
+| Action chains | ``Salt``
+| Software crash reporting | icon:times[role="danger"]
+| Staging (pre-download of packages) | ``Salt``
+| Duplicate package reporting | ``Salt``
+| CVE auditing (requires CVE ID) | ``Salt``
+| SCAP auditing | ``Salt``
+| Package verification | icon:times[role="danger"]
+| Package locking | icon:times[role="danger"]
+| System locking | icon:times[role="danger"]
+| System snapshot | icon:times[role="danger"]
+| Configuration file management | ``Salt``
+| Snapshots and profiles | ``Salt``: Profiles supported, Sync not supported
+| Power management | ``Salt``
+| Monitoring | ``Salt``
+| Docker buildhost | icon:times[role="danger"]
+| Build Docker image with OS | icon:times[role="danger"]
+| Kiwi buildhost | icon:times[role="danger"]
+| Build Kiwi image with OS | icon:times[role="danger"]
```

Comment:
Actually, Kiwi is able to build RHEL-clones images if using a RHEL-clone build host. Same for Ubuntu images.

More details here:
https://osinside.github.io/kiwi/building.html#supported-distributions

@aaannz should we update this table and say "supported" with a note to check the table I linked above to carefully choose the build host for the image?

Reply:
We should test it first. Kiwi itself may support it, but there may be differences in the image inspection, naming, etc.

I am assuming you are talking about general image building and not retail. In retail there may be even more surprises.